theano/sandbox/linalg/ops.py

```diff
@@ -280,8 +244,6 @@ def transinv_to_invtrans(node):
 @register_stabilize
 @local_optimizer([Dot, Dot22])
 def inv_as_solve(node):
-    if not imported_scipy:
```

Comment:
The call to `solve(...)` should fail if `scipy` is not present, and the `Solve` node that it builds actually needs `scipy`, see [here](https://github.com/shawntan/Theano/blob/f1f1cc087ff1a10896240ee566b4e4b155353ef7/theano/tensor/slinalg.py#L199-L200).
Did that show up as a flake8 violation?